server/apiRoutes/comments.js

```diff
@@ -0,0 +1,127 @@
+const router = require('express').Router();
+const uuid = require('uuid/v4');
+const { Comment } = require('../db/index');
+const { isLoggedIn } = require('../../utils/backend');
+
+// type: post / event
+router.get('/:type/:id', async (req, res, next) => {
+  try {
+    let { offset } = req.query;
+    let { type, id } = req.params;
+
+    if (!type || !id) {
+      res.sendStatus(400);
+    }
+
+    offset = offset ? parseInt(offset, 10) * 30 : 0;
+
+    const query = {
+      where: { [type + 'Id']: id },
+      limit: 30,
+      include: [{ model: Comment, attributes: ['id'] }],
+    };
+
```

Comment:
If we limit to 30 on the initial offset, maybe we make sure and get the last 30 created. Or the 30 most relevant according to time.